src/components/Single-Article.jsx

```diff
@@ -46,7 +84,38 @@ export const SingleArticle = () => {
                 <button disabled={voteChanges[article_id] === 1} onClick={() => handleVote(article_id, 1)}>+</button>
                 <p>Votes: {article.votes + voteChanges[article_id] || 0}</p>
                 <button disabled={voteChanges[article_id] === -1} onClick={() => handleVote(article_id, -1)}>-</button>
-                </div> 
+                <br></br><br></br>
+                <form onSubmit={handleComment}>
```

Comment:
consider abstracting this all out to its own component for a clearer separation of concerns